src/v/debug_bundle/tests/debug_bundle_service_test.cc

```diff
@@ -378,3 +378,103 @@ TEST_F_CORO(debug_bundle_service_started_fixture, termiate_never_ran) {
       term_res.assume_error().code(),
       debug_bundle::error_code::debug_bundle_process_never_started);
 }
+
+TEST_F_CORO(debug_bundle_service_started_fixture, get_file_path) {
+    using namespace std::chrono_literals;
+    debug_bundle::job_id_t job_id(uuid_t::create());
+
+    auto res = co_await _service.local().initiate_rpk_debug_bundle_collection(
+      job_id, {});
+    ASSERT_FALSE_CORO(res.has_failure()) << res.as_failure().error().message();
```

Comment:
Just a small consistency issue: Sometimes you use `!has_failure()`, which checks `!has_exception() && !has_error()`, it may be worth checking `ASSERT_TRUE(has_value()`), to ensure there's not some hiding exception.

Sometimes you use `<< res.as_failure().error().message()` and sometimes `<< res.assume_error().error().message()`, which are essentially identical.